modules/ROOT/pages/bare-metal.adoc

```diff
@@ -177,28 +177,60 @@ Alternatively, instead of generating per-machine ISOs, you can have a `--pre-ins
 === Installing on iSCSI
 
 To install CoreOS on an iSCSI boot device, follow the same
-steps as described above to get the live environnement, simply adding the correct kernel arguments.
+steps as described above to get the live environnement, and add add the iSCSI related kernel arguments.
 
 
-- Mount the iscsi target from the live environnement :
+- Mount the iSCSI target from the live environment:
 [source, bash]
 ----
 sudo iscsiadm -m discovery -t st -p 127.0.0.1
 sudo iscsiadm -m node -T iqn.2023-10.coreos.target.vm:coreos -l
 ----
 
-- run `coreos-installer`:
+- Append the necessary kargs when running `coreos-installer`:
+
+.Installing to an iSCSI target with iBFT
 [source, bash]
-----
+---
+sudo coreos-installer install \
+    /dev/disk/by-path/ip-127.0.0.1\:3260-iscsi-iqn.2023-10.coreos.target.vm\:coreos-lun-0 \
+    --append-karg rd.iscsi.firmware=1 --append-karg ip=ibft \
+    --console ttyS0 \
+    --ignition-url https://example.com/example.ign
+---
+
+.Installing to an iSCSI target with manual configuration
+[source, bash]
+---
 sudo coreos-installer install \
     /dev/disk/by-path/ip-127.0.0.1\:3260-iscsi-iqn.2023-10.coreos.target.vm\:coreos-lun-0 \
     --append-karg rd.iscsi.initiator=iqn.2024-02.com.yourorg.name:lun0 \
     --append-karg netroot=iscsi:iqn.2023-10.coreos.target.vm:coreos` \
     --console ttyS0 \
     --ignition-url https://example.com/example.ign
+---
+
+
+NOTE: All this can also be  set using `coreos-installer iso customize` or `coreos-installer pxe customize`. (See "Customizing installation" section above).
+
+For example using iBFT:
```

Comment:
Nice! I like that we provide this example. Were you able to test this?

I guess logically this goes together with what we wrote in `NOTE` but I don't think it'll appear in the same box, does it? In that case, probably cleaner to just not make it a `NOTE` anymore.

Reply:
let's try it out during test days ? 
TBH I did not spend the time to fire up the whole iscsi test setup to verify, since the test setup we have in kola works, and this is pretty straight forward.
Let's mark it to give it an actual test run during the FCOS test wek ?